Security/Guidelines/OpenSSH.mediawiki

```diff
@@ -74,6 +74,13 @@ PermitRootLogin No
 UsePrivilegeSeparation sandbox
 </source>
 
+File: <code>/etc/ssh/moduli</code>
+
+All Diffie-Hellman moduli in use should be at least 3072-bit-long (they are used for <code>diffie-hellman-group-exchange-sha256</code>). From the structure of <code>moduli</code> files, this means the fifth field of all lines in this file should be greater than or equal to 3071.
+
+To deactivate short modules in two commands: <code>awk '{if ($5 < 3071) {print ("#", $0)} else print $0}' /etc/ssh/moduli > /etc/ssh/moduli.tmp; mv /etc/ssh/moduli.tmp /etc/ssh/moduli</code>
```

Comment:
For the record, 3071 is not a typo (I have no idea why but sizes seems to be indicated minus 1…)